### DISCUSSION TITLE
conf basic motionplus

### DISCUSSION THREAD

**Discussion by gtbyio** [Motion-Project/motionplus#53] [General]
Hello

motionplus.conf 

a basic example of the configuration file for a usb webcam would be a good thing.
yes I can't display an image in the interface.

the webcam is functional 

`v4l2-ctl --list-devices
Amlogic Video Decoder (platform:meson-vdec):
	/dev/video4

Arducam USB Camera: Arducam USB (usb-c9100000.usb-1.1):
	/dev/video0
	/dev/video1
	/dev/video2
	/dev/video3
	/dev/media0
`

`lsmod | grep uvcvideo
uvcvideo              110592  0
videobuf2_vmalloc      20480  1 uvcvideo
videobuf2_v4l2         32768  3 meson_vdec,uvcvideo,v4l2_mem2mem
videobuf2_common       57344  7 meson_vdec,videobuf2_vmalloc,videobuf2_dma_contig,videobuf2_v4l2,uvcvideo,v4l2_mem2mem,videobuf2_memops
videodev              245760  5 meson_vdec,videobuf2_v4l2,uvcvideo,videobuf2_common,v4l2_mem2mem
mc                     65536  6 videodev,snd_usb_audio,videobuf2_v4l2,uvcvideo,videobuf2_common,v4l2_mem2mem
`

**Comment by Mr-Dave** [comment]
It is done by specifying like this:

`v4l2_device /dev/videoX`

I'll note to add more specificity to the the example and/or guide.
